### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by abroberts5**

### CODE REVIEW

**Comment by abroberts5** [comment]
It's 3 o'clock :(

File: house_test.rb [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+class HouseTest < MiniTest::Test
+  def test_house_class_exists
+    house = House.new("$400000", "123 sugar lane")
+    assert_instance_of House, house
+  end
+
+  def test_price_and_address_method_takes_argument
+    house = House.new("$400000", "123 sugar lane")
+    assert_equal "$400000", house.price
```
Comment (BrianZanti) [inline house_test.rb]: Take another look at the spec. The price should come back as an integer.

File: house_test.rb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+  def test_new_rooms_print_into_empty_array
+    house = House.new("$400000", "123 sugar lane")
+    room = Room.new(:bedroom, 10, 13)
+    room_1 = Room.new(:bedroom, 10, 13)
+    room_2 = Room.new(:bedroom, 11, 15)
+
+    assert_equal [room_1], house.add_room(room_1)
+    assert_equal [room_1, room_2], house.add_room(room_2)
+    #Conscious SALAD!!!
```
Comment (BrianZanti) [inline house_test.rb]: 😂

File: house_test.rb [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    room = Room.new(:bedroom, 10, 13)
+    room_1 = Room.new(:bedroom, 10, 13)
+    room_2 = Room.new(:bedroom, 11, 15)
+    room_3 = Room.new(:living_room, 25, 15)
+    room_4 = Room.new(:basement, 30, 41)
+
+    assert_equal [room_1], house.add_room(room_1)
+    assert_equal [room_1, room_2], house.add_room(room_2)
+    assert_equal [room_1, room_2, room_3], house.add_room(room_3)
+    assert_equal [room_1, room_2, room_3, room_4], house.add_room(room_4)
```
Comment (BrianZanti) [inline house_test.rb]: At some point, you also need to assert that `house.rooms` returns the correct array. It's just like when we did the Potluck challenge, when we do `add_dish` it changed our dishes array

File: house.rb [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    @rooms    = []
+  end
+
+  def add_room(new_room)
+    @rooms << new_room
+  end
+
+  def rooms_from_category(category)
+    @rooms.find_all do |room|
+      room.category
```
Comment (BrianZanti) [inline house.rb]: This is really close! `find_all` needs a boolean to know if its found something or not, but `room.category` is going to give you a symbol. You need to do `room.category == category` to compare it to the category we're looking for, i.e. the one passed in as an argument

**Pull request closed without merging**